new-integration-tests/src/test/java/oracle/weblogic/kubernetes/actions/impl/primitive/Kubernetes.java

```diff
@@ -1245,37 +1347,141 @@ public static boolean deleteService(String name, String namespace) {
     } else {
-      logger.warning("Failed to list jobs, status code {0}", list.getHttpStatusCode());
+      logger.warning("Failed to list services, status code {0}", list.getHttpStatusCode());
       return null;
     }
   }
 
+  // --------------------------- jobs ---------------------------
+
+
+  /**
+   * Delete job.
+   *
+   * @param namespace name of the namespace
+   * @param name name of the job
+   * @return true if delete is successful
+   * @throws ApiException when delete job fails
+   */
+  public static boolean deleteJob(String namespace, String name) throws ApiException {
+    try {
+      BatchV1Api apiInstance = new BatchV1Api(apiClient);
+      apiInstance.deleteNamespacedJob(
+          name, // String | name of the job.
+          namespace, // String | name of the namespace.
+          PRETTY, // String | pretty print output.
+          null, // String | When present, indicates that modifications should not be persisted.
+          GRACE_PERIOD, // Integer | The duration in seconds before the object should be deleted.
+          null, // Boolean | Deprecated: use the PropagationPolicy.
+          FOREGROUND, // String | Whether and how garbage collection will be performed.
+          null // V1DeleteOptions.
+      );
+    } catch (ApiException apex) {
+      logger.warning(apex.getResponseBody());
+      throw apex;
+    }
+    return true;
+  }
+
+  /**
+   * List jobs in the given namespace.
+   *
+   * @param namespace in which to list the jobs
+   * @return V1JobList list of {@link V1Job} from Kubernetes cluster
+   * @throws ApiException when list fails
+   */
+  public static V1JobList listJobs(String namespace) throws ApiException {
+    V1JobList list;
+    try {
+      BatchV1Api apiInstance = new BatchV1Api(apiClient);
+      list = apiInstance.listNamespacedJob(
+          namespace, // String | name of the namespace.
+          PRETTY, // String | pretty print output.
+          ALLOW_WATCH_BOOKMARKS, // Boolean | allowWatchBookmarks requests watch events with type "BOOKMARK".
+          null, // String | The continue option should be set when retrieving more results from the server.
+          null, // String | A selector to restrict the list of returned objects by their fields.
+          null, // String | A selector to restrict the list of returned objects by their labels.
+          null, // Integer | limit is a maximum number of responses to return for a list call.
+          RESOURCE_VERSION, // String | Shows changes that occur after that particular version of a resource.
+          TIMEOUT_SECONDS, // Integer | Timeout for the list/watch call.
+          Boolean.FALSE // Boolean | Watch for changes to the described resources
+      );
+    } catch (ApiException apex) {
+      logger.warning(apex.getResponseBody());
+      throw apex;
+    }
+    return list;
+  }
+
   // --------------------------- replica sets ---------------------------
+
+
+  /**
+   * Delete replica set.
```

Comment:
delete a replica set

Reply:
curious as to where we need this - i don't think we use any rs's?

Reply:
[speriyat@speriyat-1 weblogic-kubernetes-operator]$ k get rs --all-namespaces | grep opns
twooptwodomain-opns-1   weblogic-operator-69fd5c765    1         1         1       54m
twooptwodomain-opns-3   weblogic-operator-6dbccd7b95   1         1         1       51m